invenio_alma/ext.py

```diff
@@ -21,10 +22,25 @@ def __init__(self, app=None):
     def init_app(self, app):
         """Flask application initialization."""
         self.init_config(app)
+        self.init_services(app)
         app.extensions["invenio-alma"] = self
 
     def init_config(self, app):  # pylint: disable=no-self-use
         """Initialize configuration."""
         for k in dir(config):
             if k.startswith("INVENIO_ALMA_"):
                 app.config.setdefault(k, getattr(config, k))
+
+    def init_services(self, app):  # pylint: disable=no-self-use
+        """Initialize service."""
+        configuration = AlmaServiceConfig.build(app)
+        self.alma_service = (  # pylint: disable=attribute-defined-outside-init
```

Comment:
please use disable-next instead of disable, then you can write

```python
# pylint: disable-next=attribute-defined-outside-init
self.alma_service = AlmaService(config=configuration)
```

Reply:
Done